client/src/components/OnboardingPills/OnboardingPills.css

```diff
@@ -0,0 +1,19 @@
+.OnboardingPills {
+  display: inline-block;
+  width: 34.6rem;
+}
+.OnboardingPills__inner-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+.OnboardingPills .Pill {
+  width: 9.75rem;
+  height: 2.45rem;
+  font-size: 1.25rem;
+  margin: 0.5rem;
+  padding: 0;
+  box-shadow: none;
+  line-height: 2;
+  user-select: none;
+}
```

Comment:
All this CSS should be part of the `Pill` component - if you need to change something, then you should make the changes there!